src/main/kotlin/net/ccbluex/liquidbounce/features/module/ClientModule.kt

```diff
@@ -132,17 +132,25 @@ open class ClientModule(
      * If the module is running and in game. Can be overridden to add additional checks.
      */
     override val running: Boolean
-        get() = super.running && inGame && enabled
+        get() = (super.running && inGame && enabled) || disableActivation
```

Comment:
Why?

Reply:
I think it would make more sense to have it (enabled || disableActivation), because usually we would want to have the logic from before running.

Reply:
Yes, but why should a module be considered running if it cannot be enabled?

If something cannot be enabled, it is disabled

Reply:
This is an event listener, which, like ModuleBaritone, is used as an EventListener 4 RotationManager, if the module is disabled, the rotation will not be counted until the module is turned on.

Reply:
Valid point